package.json

```diff
@@ -14,6 +14,9 @@
     "type": "git",
     "url": "git://github.com/themasch/node-ebml.git"
   },
+  "engines": {
+    "node": ">= 8"
```

Comment:
I think we should be able to go with node >= 6.4 here. At least I have not seen any features not supported in node 6.4 we used until now and it is still in maintenance until mid next year.